src/components/NcBreadcrumb/NcBreadcrumb.vue

```diff
@@ -41,11 +41,8 @@ This component is meant to be used inside a Breadcrumbs component.
 		@dragleave="dragLeave">
 		<component :is="tag"
 			v-if="(nameTitleFallback || icon) && !$slots.default"
-			:exact="exact"
-			:to="to"
-			:href="href"
 			:title="title"
-			v-bind="$attrs"
+			v-bind="{ ...$attrs, ...to && { to, exact }, ...!to && { href } }"
 			v-on="$listeners">
```

Comment:
This expression seems to be too complicated to me for template. Especially spreading boolean values.

What about making a computed property with `if` statement?